integration/env.go

```diff
@@ -184,9 +184,10 @@ func (e Env) Scaffold(name string, flags ...string) (appPath string) {
 
 	appDir := path.Base(name)
 
-	// Cleanup the home directory of the app
+	// Cleanup the home directory and cache of the app
 	e.t.Cleanup(func() {
 		os.RemoveAll(filepath.Join(e.Home(), fmt.Sprintf(".%s", appDir)))
+		os.RemoveAll(filepath.Join(e.Home(), fmt.Sprintf(".ignite/cache/%s", appDir)))
```

Comment:
Not sure how to avoid this. 
It looks like the tests already to this on the line above too?

My thinking was that since it is only the cache for the test chains and the worst case scenario is someone has a chain named the same it will be slower on one execution, I thought it wasn't too harmful?

Reply:
You're right. I created an issue for that one: https://github.com/ignite-hq/cli/issues/2459

I think it's OK for now, we can address this later in a separate issue. 

But I beleive we should be able to pass a different root for the storage in the chain pkg so, the `cache` pkg won't depend on `chainconfig`. If we make this configurable, we could just choose a different root (a temp dir) in the tests and it'll address this point as well?

Reply:
I changed the cache package to do this, but I mainly just moved the `chainconfig` call one level up to the service layer instead. Take a look and let me know if you would like that done differently.